src/map/map.ts

```diff
@@ -335,6 +335,7 @@ export class PropertiesMap {
         if (this._is3D()) {
             data.marker.style.display = 'none';
         } else {
+            this._active = guid;
```

Comment:
It is strange that this is required, `_updateSelectedMarker` should be able to update even non-active markers. I'll have a quick look.